keras/backend/tensorflow_backend.py

```diff
@@ -2686,7 +2686,11 @@ def _legacy_call(self, inputs):
                     (np.expand_dims(sparse_coo.row, 1),
                      np.expand_dims(sparse_coo.col, 1)), 1)
                 value = (indices, sparse_coo.data, sparse_coo.shape)
-            feed_dict[tensor] = value
+            if type(tensor) is list:
```

Comment:
Use `isinstance` instead of `type`

Reply:
I add two commits, one is to create a file to test multi inputs, the other is to change type() to isinstance()

Reply:
> Use `isinstance` instead of `type`

I add two commits, one is to create a file to test multi inputs, the other is to change type() to isinstance()